sdk/identity/identity/src/credentials/environmentCredential.ts

```diff
@@ -125,7 +125,7 @@ export class EnvironmentCredential implements TokenCredential {
    * @param options - Optional parameters. See {@link GetTokenOptions}.
    */
   async getToken(scopes: string | string[], options: GetTokenOptions = {}): Promise<AccessToken> {
-    return trace(`${credentialName}.getToken`, options, async (newOptions) => {
+    return tracingClient.withSpan(`${credentialName}.getToken`, options, async (newOptions) => {
```

Comment:
I noticed you use `credentialName` here instead of `this.constructor.name` - is there a reason for the difference?

Reply:
The real reason for me to use it in this case, at least when I wrote this code, was because in this file I’m referencing it a bit too many times, so `credentialName` is shorter 🙂